### PR TITLE
Create separate module for WebGL constants

### PR DESCRIPTION
**Pull request by ibgreen**
For #

#### Background
- Follows up on the module split started in v6.x

#### Change List
- Create a separate module for the WebGL constants. 
- Once landed, will phase out the `import luma.gl/constants` hack and keep this large file separated from the core lib.

### CODE REVIEW

**Comment by Pessimistress** [comment]
We have a large number of `import GL from 'luma.gl/constants'` in deck.gl and applications. What is the upgrade path for them?

File: modules/core/constants.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line
-module.exports = require('./dist/es5/constants');
+console.warn('"import `luma.gl/constants`"" has been replaced with "import `@luma.gl/constants`"');
+require('@luma.gl/connstants');
```
Comment (Pessimistress) [inline modules/core/constants.js]: - Typo
- Still need to export it?

Comment (Pessimistress) [reply modules/core/constants.js]: Shouldn't this be added to the dependencies of the core package?

Comment (ibgreen) [reply modules/core/constants.js]: Good point, done.

**Pull request merged**